### ISSUE TITLE
View's `measure` + other native methods are null

### ISSUE THREAD

**Issue by bgneu** [testing-library/native-testing-library#129]
- `react-native` or `expo`: react-native
- `native-testing-library` version: 5.0.3
- `jest-preset`: @testing-library/react-native/jest-preset
- `react-native` version: 0.62.2
- `node` version: 14.4.0

### Relevant code or config:

```js
_updatePosition(callback) {
    if (this._button && this._button.measure) {
      this._button.measure((fx, fy, width, height, px, py) => {
        this._buttonFrame = {
          x: px,
          y: py,
          w: width,
          h: height,
        };
        callback && callback();
      });
    }
  }
```
`this._button` is a ref to `TouchableOpacity` component.

### What you did:

Upgraded existing project from RN 0.61.5, where I had this reported [issue](https://github.com/testing-library/native-testing-library/issues/125), to 0.62.2.

With the upgrade, however, it seems like the [mock-native-methods](https://github.com/testing-library/native-testing-library/blob/master/src/preset/mock-native-methods.js) are no longer accessible because on close inspection`this._button.measure === null` returns true on tests. This is also true for the [other native methods](https://reactnative.dev/docs/direct-manipulation#other-native-methods).

### What happened:

The `measure` function is not available, therefore not being called. 

### Reproduction:

I'll try to update the issue with a repo that reproduces the issue.

### Problem description:

The `measure` method, along with all other native methods mentioned in [RN docs](https://reactnative.dev/docs/direct-manipulation#other-native-methods) should be available on most of the default components provided by React Native.

### Suggested solution:

Attempted to look into the `@testing-library/react-native` preset code, but could not spend too much time as to come up with a solution or improvement.

### Can you help us fix this issue by submitting a pull request?

Would love to, but I have some other work priorities which may not allow me to devote the needed time to fix this. If it changes before this is resolved I'll definitely look into it.

**Comment by bgneu** [comment]
Closing this issue, because as I debugged a bit more the issue is with the Touchable components themselves. In order to get the press fireEvent to work we have to mock `TouchableOpacity` (see https://github.com/testing-library/native-testing-library/issues/113#issuecomment-607796505), however once mocked, attempting to get a reference to the component does not work, which then snowballs into the issue that caused me to create this ticket.

In short I believe this is related to https://github.com/testing-library/native-testing-library/issues/113#issuecomment-607796505. On 0.61.5 I was still able to get a ref to the component however, and it seems like upgrading to 0.62.2 further aggravated the issue.

A work around that's working for me now is 
```
jest.mock('react-native/Libraries/Components/Touchable/TouchableOpacity', () => {
  const React = require('react')
  const { View } = jest.requireActual('react-native')
  class TouchableOpacity extends React.Component {
    measure = jest.fn().mockImplementation((callback) => {
      callback(0, 0, 100, 100, 10, 10) // Random values
    })
    render() {
      return <View {...this.props}/>
    }
  }

  TouchableOpacity.displayName = 'TouchableOpacity';
  return TouchableOpacity;
});
```
